go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/go-redis/redis v6.15.6+incompatible
 	github.com/hewigovens/go-coincodec v1.0.4
 	github.com/jinzhu/gorm v1.9.12
+	github.com/magiconair/properties v1.8.1
```

Comment:
any reason to include this dependency? we have a good testing dependency: `github.com/stretchr/testify`

Reply:
No reason, my fault, will be removed

Reply:
can you remove that? please